tests/test-mongoc-collection.c

```diff
@@ -1454,7 +1454,7 @@ test_aggregate_legacy (void *data)
 
    /* no "cursor" argument */
    request = mock_server_receives_command (
-      server, "db", MONGOC_QUERY_NONE,
+      server, "db", MONGOC_QUERY_SLAVE_OK,
```

Comment:
Yeah, I think you fixed an unreported bug - "aggregate" should've sent slave ok the whole time.